api/e2etests/e2e_seed_data.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from environments.models import Environment
+from organisations.models import Organisation, OrganisationRole
+from projects.models import Project
+from users.models import FFAdminUser
+
+
+class E2ESeedData:
+    fe_e2e_test_user_email = (
+        os.environ["FE_E2E_TEST_USER_EMAIL"]
+        if "FE_E2E_TEST_USER_EMAIL" in os.environ
+        else None
+    )
+    password = "str0ngp4ssw0rd!"
+
+    def teardown(self):
+        # delete users created for e2e test by front end
+        if self.fe_e2e_test_user_email:
+            FFAdminUser.objects.filter(email=self.fe_e2e_test_user_email).delete()
+            FFAdminUser.objects.filter(
+                email=self.fe_e2e_test_user_email + ".io"

```

Comment:
It's a bit odd to me that we're adding ".io" here, couldn't we just have [test@example.com](mailto:test@example.com) and [test2@example.com](mailto:test2@example.com) or something?

Reply:
Ok, I will do it

Reply:
Done